compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -231,13 +276,11 @@ def get_advice(self, state: time_step.TimeStep) -> np.ndarray:
     return policy_action
 
 
-class ExplorationWorker(worker.Worker):
+class ExploreModule:
```

Comment:
make it a noun: `ModuleExplorer`

Reply:
Forgot about that, sorry. Going to fix it now.